README.md

```diff
@@ -1,2 +1,21 @@
-# documentation
-Official Yield Swap Documentation
+# Introduction
+
+## What is Yield Swap?
+
+Yield Swap is a decentralized exchange that allows users to swap interest rates.
+Users exchanging interest rates on Yield Swap are called [Payers](./payers.md)
+and [Receivers](./receivers.md). Payers pay the fixed interest rate of an asset
+in exchange for the floating interest rate provided by Receivers. Both user
+types do this in order to make a profit based on their respective investment
+strategies.
+
+## What problem does Yield Swap solve?
+
```

Comment:
The core problem we solve is that we fill an important "risk transformation" gap: i.e. currently DeFi is lacking a market for transferring yield risks.

The absence of such a market also means that there's no way to reflect future yield expectations. 

Main use cases for yield swap are: (1) speculation and (2) hedging.

Reply:
I tried to incorporate your suggestions.